sw/device/lib/dif/autogen/dif_gpio_autogen_unittest.cc

```diff
@@ -193,17 +173,14 @@ TEST_F(IrqForceTest, NullArgs) {
 }
 
 TEST_F(IrqForceTest, BadIrq) {
-  EXPECT_EQ(dif_gpio_irq_force(nullptr, (dif_gpio_irq_t)32), kDifBadArg);
+  EXPECT_EQ(dif_gpio_irq_force(nullptr, static_cast<dif_gpio_irq_t>(32)),
+            kDifBadArg);
 }
 
 TEST_F(IrqForceTest, Success) {
   // Force first IRQ.
   EXPECT_WRITE32(GPIO_INTR_TEST_REG_OFFSET, {{0, true}});
   EXPECT_EQ(dif_gpio_irq_force(&gpio_, kDifGpioIrqGpio0), kDifOk);
-
-  // Force last IRQ.
-  EXPECT_WRITE32(GPIO_INTR_TEST_REG_OFFSET, {{31, true}});
-  EXPECT_EQ(dif_gpio_irq_force(&gpio_, kDifGpioIrqGpio31), kDifOk);
```

Comment:
nit: Was this removed because GPIO IRQs are defined as a single field? It looks like the test was generated as expected before this change. Would you mind taking a look if a small tweak can bring back these?

Reply:
Ah yes, another great catch! Fixed the template, and re-gen'd.

Reply:
Thanks!